docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+version: "3.7"
+
+networks:
+  helium:
+
+services:
+  gateway-rs:
+    image: helium/gateway-rs:v1.0.0-alpha.23
+    build:
+      context: https://github.com/helium/gateway-rs.git#v1.0.0-alpha.23
```

Comment:
How are we going to keep this up to date?

Reply:
We can change the tag to latest and just have it build main

Reply:
I changed it to just build the local source.